debian/usr.bin.snap-confine

```diff
@@ -68,29 +68,38 @@
     # change_profile -> **,
 
     # reading seccomp filters
-    /var/lib/snapd/seccomp/profiles/* r,
+    /{tmp/snap.rootfs_*/,}var/lib/snapd/seccomp/profiles/* r,
 
     # set up snap-specific private /tmp dir
     capability chown,
-    /tmp/ w,
-    /tmp/snap.*/ w,
-    /tmp/snap.*/tmp/ w,
-    mount options=(rw private) -> /tmp/,
-    mount options=(rw bind) /tmp/snap.*/tmp/ -> /tmp/,
-    mount fstype=devpts options=(rw) devpts -> /dev/pts/,
-    mount options=(rw bind) /dev/pts/ptmx -> /dev/ptmx,   # for bind mounting
-
-    # for running snaps on ubuntu
+    /{tmp/snap.rootfs_*/,}tmp/ w,
+    /{tmp/snap.rootfs_*/,}tmp/snap.*/ w,
+    /{tmp/snap.rootfs_*/,}tmp/snap.*/tmp/ w,
+    mount options=(rw private) ->  /{tmp/snap.rootfs_*/,}tmp/,
+    mount options=(rw bind) /{tmp/snap.rootfs_*/,}tmp/snap.*/tmp/ -> /{tmp/snap.rootfs_*/,}tmp/,
+    mount fstype=devpts options=(rw) devpts -> /{tmp/snap.rootfs_*/,}dev/pts/,
+    mount options=(rw bind) /{tmp/snap.rootfs_*/,}dev/pts/ptmx -> /{tmp/snap.rootfs_*/,}dev/ptmx,   # for bind mounting
+
+    # for running snaps on classic
     mount options=(rw rslave) -> /,
-    /snap/ r,
-    /snap/** r,
-    mount options=(rw bind) /snap/ubuntu-core/*/bin/ -> /bin/,
-    mount options=(rw bind) /snap/ubuntu-core/*/sbin/ -> /sbin/,
-    mount options=(rw bind) /snap/ubuntu-core/*/lib/ -> /lib/,
-    mount options=(rw bind) /snap/ubuntu-core/*/lib32/ -> /lib32/,
-    mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
-    mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
-    mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    /{tmp/snap.rootfs_*,}snap/ r,
+    /{tmp/snap.rootfs_*,}snap/** r,
+    mount options=(rw bind) /snap/ubuntu-core/*/ -> /tmp/snap.rootfs_*/,
```

Comment:
Can you add a comment for this and why it is different than the next group or rules (you can add a comment there too).